operator/apis/lifecycle/v1alpha2/keptnworkload_types.go

```diff
@@ -102,10 +102,18 @@ func (w KeptnWorkload) GenerateWorkloadInstance(previousVersion string, traceCon
 	}
 }
 
-func (i KeptnWorkload) GetSpanAttributes() []attribute.KeyValue {
+func (w KeptnWorkload) GetSpanAttributes() []attribute.KeyValue {
 	return []attribute.KeyValue{
-		common.AppName.String(i.Spec.AppName),
-		common.WorkloadName.String(i.Name),
-		common.WorkloadVersion.String(i.Spec.Version),
+		common.AppName.String(w.Spec.AppName),
```

Comment:
afaik, `AppVersion` is always defined. Hence, I would suggest to add also this info